wls-exporter-core/src/main/java/com/oracle/wls/exporter/ServletInvocationContext.java

```diff
@@ -31,7 +31,9 @@ public void close() {
 
   @Override
   public UrlBuilder createUrlBuilder() {
-    return UrlBuilder.create(request.getLocalName(), request.isSecure())
+    return UrlBuilder.create(request.isSecure())
+          .withHostName(request.getLocalName())
+          .withHostName("localhost")
```

Comment:
if I understand correctly, we will always try localhost if the connection using request.getLocalName() failed. Should this be documented somewhere?

Reply:
Yes, it should - and it should not be using a literal string. I will fix it.